Resources/Prototypes/_NF/Entities/Structures/Shuttles/cannons.yml

```
@@ -13,3 +14,32 @@
   components:
   - type: Machine
     board: Null
+
+# Mail
+- type: entity
+  id: ShuttleGunMailLauncher
+  parent: [ ShuttleGunFriendship ]
+  name: stationary mail launcher
+  description: You've got mail! A small stationary mail launcher that holds 2 grenades.
+  components:
+  - type: Sprite
+    sprite: _NF/Objects/Weapons/Guns/Shuttles/maillauncher.rsi
+    layers:
+    - state: mail-launcher
+    - state: mag-2
+      map: ["enum.GunVisualLayers.Mag"]
+  - type: Gun
+    projectileSpeed: 30
```

```suggestion
    projectileSpeed: 30
    defaultDirection: 0, -30 # Target 30 m in front of the cannon in case it's indoors.
```
Because this is throwing the item vs. firing it (I think?), set a longer default direction to ensure it flies downrange if indoors.

---

duped